README.md

```diff
@@ -14,6 +14,14 @@ A document and demo of the DRA support for GPUs provided by this repo can be fou
 
 This section describes using `kind` to demo the functionality of the NVIDIA GPU DRA Driver.
 
+Here's a demo showing how to install and configure DRA, and run a pod in a `kind` cluster on a Linux workstation. 
+
+<p align="center">
+<img width="800" src="./demo/specs/quickstart/basic-demo.svg">
+</p>
+
+Below are the detailed, step-by-step instructions.
```

Comment:
I think this is too high in the README now. It should be after the prerequisites, but before the instructions for creating the kind cluster.

Reply:
My bad. What you suggested was my intention. I made a mistake 😢. Fixed it.

Reply:
Does it look good?